packages/date-time-picker/src/DateTimePicker.ts

```diff
@@ -0,0 +1,158 @@
+/*
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import {
+    CSSResultArray,
+    html,
+    TemplateResult,
+} from '@spectrum-web-components/base';
+import {
+    property,
+    state,
+} from '@spectrum-web-components/base/src/decorators.js';
+import { InputSegments } from '@spectrum-web-components/input-segments';
+
+import styles from './date-time-picker.css.js';
+
+import '@spectrum-web-components/calendar/sp-calendar.js';
+import '@spectrum-web-components/field-label/sp-field-label.js';
+import '@spectrum-web-components/icons-workflow/icons/sp-icon-calendar.js';
+import '@spectrum-web-components/overlay/sp-overlay.js';
```

Comment:
Please add a note to the to do lists that are tracked around adding lazy loading for some of these dependancies.

Reply:
What do you mean exactly, @Westbrook?